concepts.mdx

```diff
@@ -141,18 +154,54 @@ This is an extremely powerful feature of Flipt that can help you seamlessly
 deploy new features of your applications to your users while also limiting the reach
 of potential bugs.
 
+## Rollouts
+
+<Info>
+  Rollouts are only available for [Boolean](#boolean) flags. Rollouts were
+  introduced in v1.24.0 of Flipt.
+</Info>
+
+Rollouts allow you to configure boolean flags to return `true` or `false` for a given flag given a set of rollout rules.
```

Comment:
Here is just another way of wording for your consideration.
My thought here is to potentially remove the need for the `Note` below.

I pause for thought around the wording of default being tied to enabled.

In my head the flags returned value is enabled, but this return value can be overriden by a matching rollout rule.

```
Boolean flags return `true` or `false` based on the flags `enabled` property.

However, the flags return value can be changed using Rollouts.

Rollouts are a sequence of conditions which when one is matched for a request context, it overrides the flags return value.
```